automation_tools/satellite6/upgrade/capsule.py

```diff
@@ -0,0 +1,161 @@
+import os
+import sys
+import time
+
+from tools import reboot
+from automation_tools import setup_capsule_firewall
+from automation_tools.repository import enable_repos, disable_repos
+from automation_tools.satellite6.capsule import generate_capsule_certs
+from automation_tools.utils import distro_info, update_packages
+from fabric.api import env, execute, run
+from tasks import (
+    create_rhevm_instance,
+    delete_rhevm_instance,
+    sync_capsule_repos_to_upgrade
+)
+from tools import copy_ssh_key
+
+
+def satellite6_capsule_setup(sat_host, os_version):
+    """Sets up required things on upgrade running machine and on Capsule to
+    perform capsule upgrade later
+
+    :param string sat_host: Satellite hostname to which the capsule registered
+    :param string os_version: The OS version onto which the capsule installed
+        e.g: rhel6, rhel7
+    """
+    # If Personal Capsule Hostname provided
+    if os.environ.get('CAPSULE_HOSTNAMES'):
+        cap_hosts = os.environ.get('CAPSULE_HOSTNAMES')
+        # For Personal Satellite CAPSULE_SUBSCRIPTION is must
+        cap_subscription = os.environ.get('CAPSULE_SUBSCRIPTION')
+        if not cap_subscription:
+            print('CAPSULE_SUBSCRIPTION environment variable is not '
```

Comment:
Empty String i.e(''), that equals to false in python. So if not defined it('') will be treated as false. And it is been tested already :)